m2e-maven-runtime/pom.xml

```diff
@@ -35,7 +35,7 @@
 
 	<properties>
 		<maven-bundle-plugin.version>4.2.1</maven-bundle-plugin.version>
-		<m2e.version>1.16.1</m2e.version>
+		<m2e.version>1.16.0</m2e.version>
```

Comment:
Unnecessary change, causing errors.

Reply:
> Unnecessary change, causing errors.

I think this came in one of the times I rebased. I believe when I submitted the PR originally this was the version and the build didn't work with the same errors at the time.